kolibri/plugins/facility/assets/src/modules/pluginModule.js

```diff
@@ -48,7 +48,7 @@ export default {
       // parameter if we're in a multi-facility situation
       const params = {};
       if (getters.userIsMultiFacilityAdmin) {
-        params.facility_id = getters.activeFacilityId;
+        params.facilityId = getters.activeFacilityId;
```

Comment:
The route uses the parameter `facility_id` below which is what this is encoding, so you should revert this change.